modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Sequence.scala

```diff
@@ -12,6 +13,10 @@ case class Sequence[+A](id: String, metadata: SequenceMetadata, steps: List[Step
 
 object Sequence {
 
+  type Id = String
```

Comment:
I mas thinking about the `ObsId` type on `DhsClient`. Most likely though, that definition is a misnomer, and it should be more like `ImageId` right?